drivers/opamp/Kconfig

```diff
@@ -9,7 +10,7 @@ menuconfig OPAMP
 if OPAMP
 
 module = OPAMP
-module-str = opamp
+module-str = OPAMP driver
```

Comment:
given the original version is what 99.5% of the drivers use as a convention, I would not change it

Reply:
Sure, I'll do. Thank you both.

Reply:
done